Description: read packet even though it will be ignore to avoid endless call loop
Origin: backport, http://open-mesh.net/changeset/1320
Author: Marek Lindner <lindner_marek@yahoo.de>

---
diff --git a/routing.c b/routing.c
index 74e3cdb3d0f07815cb9c8a5dbfc22a7c5f2f71c7..94c6455c458a0bfe2fa3ee14320faef89ac177d4 100644
--- a/routing.c
+++ b/routing.c
@@ -617,6 +617,7 @@ int packet_recv_thread(void *data)
 			while (1) {
 				if (batman_if->if_active != IF_ACTIVE) {
 					debug_log(LOG_TYPE_NOTICE, "Could not read from deactivated interface %s!\n", batman_if->dev);
+					receive_raw_packet(batman_if->raw_sock, packet_buff, PACKBUFF_SIZE);
 					result = 0;
 					break;
 				}
